gdal/ogr/ogrsf_frmts/ngw/drv_ngw_raster.html

```diff
@@ -0,0 +1,164 @@
+Each NextGIS Web vector or PostGIS layer can have one or more vector style (QGIS
+qml or MapServer xml).<br>
+WMS layer is one or more layers from external WMS service. This layer has no
+styles.<br>
+WMS Service is usual WMS protocol implementation.
+<p>
+Now NGW driver supports only raster and vector styles and WMS layers. You can get
+raster data as tiles or image (only tiles are supported now).
+</p>
+The driver supports read-only operations on rasters.
+
+<h2>Dataset name syntax</h2>
+
+The minimal syntax to open a NGW datasource is:
+
+NGW:[NextGIS Web URL][/resource/][resource identifier]
+
+<ul>
+    <li><b>NextGIS Web URL</b> may be an url to nextgis.com cloud service (for
+        example, https://demo.nextgis.com), or some other url including port and
+        additional path (for example, http://192.168.1.1:8000/test).
+    <li><b>resource</b> is mandatory keyword dividing resource identifier from
+        the rest of URL.
+    <li><b>resource identifier</b> this is positive number from 0 and above. This
+        may be an resource group, vector, PostGIS or raster layer, style.
+</ul>
+
+If identifier is resource group, all vector layers, PostGIS, raster layers, styles
+will listed as child resources. In other case this will be a separate raster.
+
+<h2>Configuration options</h2>
+
+The following configuration options are available:
+
+<ul>
+    <li><b>NGW_USERPWD</b>: User name and password separated with colon. Optional
+        and can be set using open options.
+    <li><b>NGW_CACHE_EXPIRES</b>: Time in seconds cached files will stay valid. If
+        cached file expires it is deleted when maximum size of cache is reached.
+        Also expired file can be overwritten by the new one from web. Defaults
+        to 604800 (7 days).
+    <li><b>NGW_CACHE_MAX_SIZE</b>: The cache maximum size in bytes. If cache
+        reached maximum size, expired cached files will be deleted. Defaults
+        to 67108864 (64Mb).
+</ul>
+
+<h2>Authentication</h2>
+
+Any operations (read, write, get metadata, chage properties, etc.) may require
+an authenticated access. Authenticated access is obtained by specifying user name
+and password in open, create or configuration options.
+
+<h2>Open options</h2>
+
+Starting with GDAL 2.0, the following open options are available:
+
+<ul>
+    <li>USERPWD - Username and password, separated by colon.
+    <li>CACHE_EXPIRES=604800 - Time in seconds cached files will stay valid. If
+        cached file expires it is deleted when maximum size of cache is reached.
+        Also expired file can be overwritten by the new one from web. Defaults
+        to 604800 (7 days).
+    <li>CACHE_MAX_SIZE=67108864 - The cache maximum size in bytes. If cache
+        reached maximum size, expired cached files will be deleted. Defaults
+        to 67108864 (64Mb).
+</ul>
+
+<h2>Metadata</h2>
+
+NextGIS Web metadata are supported in datasource, vector, PostGIS, raster layers
```

Comment:
apply my previous remarks from the vector side

Reply:
Fixed per 4579aeab927d70ddab6142340bf5550c6789b42e